app/Repositories/Activity/ActivityRepository.php

```diff
@@ -195,4 +198,84 @@ public function getH5pElasticsearchFields($h5pContent)
 
         return $h5pElasticsearchFieldsArray;
     }
+
+    public function clone(Request $request, Playlist $playlist, Activity $activity) 
+    {
+        $h5P_res = Null;
```

Comment:
Use either null, or NULL.

Secondly, I see that you've not used camelCase for variables. 
I was told that H5P would be refactored in the next release, so you can leave that for now, but please note it for your record.

$h5P_res could be $h5pRes, as an example.